construct.js

```diff
@@ -11,11 +12,18 @@ module.exports = (answers, templates, overwrite) => templates.forEach((template)
 
         ensureDirectories(destination);
 
-        if (!fs.existsSync(destination) || overwrite) {
+        if (!fs.existsSync(destination) || overwrite || template.overwrite) {
```

Comment:
We can get rid of the `|| overwite`...it's better if the calling app specifies it per-template, to avoid accidentally overwrites.